### PR TITLE
Shubhusion/issue98

### LINKED ISSUE #98: Improvements to the site README.md file
The Uptane project has recently adopted [Docusaurus](https://docusaurus.io/) as its primary platform for building its website. This transition marks an exciting step forward in our project's development. 

The current readme file falls short in terms of clarity and effectiveness, which undermines our ability to effectively communicate and collaborate on our project. To rectify this, we are seeking your valuable input and suggestions to update and revamp our readme file. We believe that collective insights and ideas will help us create a more informative and user-friendly document.

We welcome any recommendations, be it for restructuring, content, or design elements that can improve the readability and usability of the readme file. Your contribution is essential in ensuring that our project is accessible and comprehensible to a wider audience.

PS: If you are interested in providing suggestions or assistance regarding the content and formatting of the readme file, please feel free to reach out to @jhdalek55 @hexsecs and @Abhijay007.

### PR DESCRIPTION
**Pull request by shubhusion**
# Summary of Changes

- Redesigned the Readme.md 
- Added Badges 
- Added Troubleshooting Tips Section Under Running
- Added Connect with Us Section 

## Related Issue

Closes #98 

## Checklist

- [x] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
- [x] My code is well-documented, and I've updated relevant documentation.
- [x] I have added or updated test cases to ensure the code's functionality.
- [x] I have tested these changes on my local environment.
- [x] All tests pass, and there are no new linting errors.
- [x] I have reviewed and proofread my code and the changes.
- [x] The branch is up-to-date with the base branch.

## Screenshots (if applicable)

Before Changes:
image

After Changes:
image

## Additional Context

Add any additional context or information that might be helpful for reviewers.

## Reviewer(s)

@hexsecs @jhdalek55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,197 @@
...
+
+## 🤝 Contributing
+
+Contributions are welcome! Here are several ways you can contribute:
+
+- **[Report Issues](https://github.com/uptane/uptane.github.io/issues)**: 🐛 Submit any bugs found or log feature requests for UPTANE. Please await assignment for your submitted issues.
+- **[Submit Pull Requests](https://github.com/uptane/uptane.github.io/blob/main/CONTRIBUTING.md)**: 🛠️  submit your PRs addressing the assigned issues.
+- **[Join the Discussions](https://github.com/uptane/uptane.github.io/discussions)**: 💬 Engage in discussions by sharing your insights, providing feedback, or asking any relevant questions.
+
+For more info please refer to Uptane [Contribution guidelines](https://github.com/uptane/uptane-standard/blob/master/CONTRIBUTING.md)
```
Comment (jhdalek55) [inline README.md:171]:
```suggestion
For more information, please refer to Uptane [Contribution guidelines](https://github.com/uptane/uptane-standard/blob/master/CONTRIBUTING.md)
```

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
Except for spelling out the word "information," this is fine with me.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,197 @@
...
+This project is managed by Prof. Justin Cappos and other members of the [Secure Systems Lab](https://ssl.engineering.nyu.edu/) at NYU.
+
+Uptane was initiated with support from the U.S. Department of Homeland Security grants D15PC00239 and D15PC00302. The views and conclusions contained herein are the authors’ and should not be interpreted as necessarily representing the official policies or endorsements, either expressed or implied, of the U.S. Department of Homeland Security (DHS) or the U.S. government.
+
+---
+
+## Connect With Us
+
+[![LinkedIn](https://img.shields.io/badge/LinkedIn-Connect-blue)](https://www.linkedin.com/company/uptane/)
+[![Discord](https://img.shields.io/discord/uWyT6gDCqx?label=Discord&logo=discord)](https://discord.gg/uWyT6gDCqx)
```
Comment (hexsecs) [inline README.md:194]: This link doesn't work. You need the server ID.
https://img.shields.io/discord/1157306344097988640?label=Discord&logo=discord

Comment (hexsecs) [inline README.md:194]:
```suggestion
[![Discord](https://img.shields.io/discord/1157306344097988640?label=Discord&logo=discord)](https://discord.gg/uWyT6gDCqx)
```

File: README.md [hunk 3]
```diff
@@ -1,70 +1,197 @@
...
-either expressed or implied, of the U.S. Department of Homeland Security (DHS)
-or the U.S. government.
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD041 -->
+<div  align="center">
+<!-- markdownlint-enable MD041 -->
+
+<h1  align="center">
+
+<img  src="https://github.com/uptane/uptane.github.io/blob/main/static/img/uptane-logo.svg"  width="500" />
```
Comment (hexsecs) [inline README.md:8]:
```suggestion
<img  src="https://github.com/uptane/uptane.github.io/blob/main/static/img/uptane_logo_light.svg"  width="500" />
```
Should use the new logo

**Review by Abhijay007 — CHANGES_REQUESTED** [review]
@shubhusion Could you please change the text 'built with' to a badge instead

image

something like this : 
image


you can use : https://forthebadge.com/#/generator for this purpose

**Comment by Abhijay007** [comment]
> @shubhusion Could you please change the text 'built with' to a badge instead
> 
> image
> 
> something like this : image
> 
> you can use : https://forthebadge.com/#/generator for this purpose

Hi @shubhusion instead of forthebadge can you please use https://shields.io/badges

**Comment by shubhusion** [comment]
> > @shubhusion Could you please change the text 'built with' to a badge instead
> > image
> > something like this : image
> > you can use : https://forthebadge.com/#/generator for this purpose
> 
> Hi @shubhusion instead of forthebadge can you please use https://shields.io/badges

image
@Abhijay007 let me know if this works?

**Comment by Abhijay007** [comment]
> > > @shubhusion Could you please change the text 'built with' to a badge instead
> > > image
> > > something like this : image
> > > you can use : https://forthebadge.com/#/generator for this purpose
> > 
> > 
> > Hi @shubhusion instead of forthebadge can you please use https://shields.io/badges
> 
> image @Abhijay007 let me know if this works?

ya it LGTM

**Comment by hexsecs** [comment]
That's perfect. If you can update to that, I can merge the PR today.

File: README.md [hunk 4]
```diff
@@ -1,70 +1,197 @@
...
+<h1  align="center">
+
+<img  src="https://github.com/uptane/uptane.github.io/blob/main/static/img/uptane_logo_light.svg"  width="500" />
+
+</h1>
+
+<p  align="center">
+
+<a href="https://docusaurus.io/">
+  Built With <img src="https://img.shields.io/badge/docusaurus-231F20.svg?style=flat-square&logo=docusaurus&logoColor=Green" alt="docusaurus" width="120"/>
```
Comment (hexsecs) [inline README.md:15]: Update to harmonized badge (see comment from @Abhijay007 )
https://github.com/uptane/uptane.github.io/pull/142#issuecomment-1816476877

**Review by hexsecs — CHANGES_REQUESTED** [review]
See comment. Should be GTG after that.

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Review by hexsecs — APPROVED** [review]

**Pull request merged**